src/pcre2grep.c

```diff
@@ -2528,6 +2528,7 @@ while (length > 0)
 necessary, otherwise assume fork(). */
 
 #ifdef WIN32
+(void)fflush(stdout);
```

Comment:
Running the Grep tests on Windows showed that they need the same fflush() fix as Unix.

I believe the Unix code is still wrong however: you're doing `fflush()` after the call to fork()! We need to flush first, before the child process starts writing anything. I'm a bit surprised that you got away with that (probably because most of the printing that's done by pcre2grep is newline-terminated, and stdout is flushed on newline usually).

You didn't need two calls to fflush either on Unix (one before and one after waitpid).